scanner/src/components/SectionHeader/index.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  export let title
+</script>
+
+<div class="title">
+  {title}
+</div>
+<div class="subtitle">
+  <slot />
```

Comment:
I also hesitate if I should use just second prop in cases like this.  I mean if text not expected to be large. But just thinking aloud again. And subtitle is almost the same as `Subtitle`, just extra margin added. Could it be reused here?

Reply:
Just to be clear we don't want to avoid adding props, we want to avoid adding props with names like `noBottomMargin`